integration_tests/tests/self_test.rs

```diff
@@ -1,10 +1,80 @@
 use std::net::{TcpListener, TcpStream};
 
-use simple_https_client::tls13client;
+use simple_https_client::{
+    tls13client, SHA256_Aes128Gcm_EcdsaSecp256r1Sha256_P256,
```

Comment:
Lots of these show up as unused when not being tested. How about using the full path for the ciphersuites instead to avoid this?